File: iShop/iShop/Model/Product.swift
//
//  Product.swift
//  iShop
//
//  Created by Deepanshu Jain on 07/06/23.
//

import Foundation

class Product {
    
    let id: Int
    let name: String
    let description: String
    let image: String
    let price : Double

    init(id: Int, name: String, description: String, image: String, price:Double) {
        self.id = id
        self.name = name
        self.description = description
        self.image = image
        self.price = price
    }
}

File: iShopAPI/iShopAPI/Model/Product.cs
using System;
namespace iShopAPI
{
	public class Product
	{
        public int id { set; get; } 
        public string? title { set; get; }
        public string? description { set; get; }
        public string? image { set; get; }
        public double price { set; get; }
        public string? category { set; get; }
    }

    public class ProductResponse
    {
        public List<Product>? products { set; get; }
    }
}

